ext/webgpu/src/lib.rs

```diff
@@ -65,6 +76,8 @@ pub mod queue;
 pub mod render_pass;
 pub mod sampler;
 pub mod shader;
+#[cfg(feature = "surface")]
```

Comment:
Why is it hidden behind a feature flag? Is this flag on by default?

Reply:
no its disabled by default. its not important directly to cli, but for other projects